docs/btc-transactions-validity.md

```diff
@@ -1,23 +1,179 @@
 ## Introduction
 
-##  Prerequisites
+## Prerequisites
 
 ## Taproot outputs
 
-## Types of transactions
+## Building Transactions
 
-## Staking
+There are three special transaction types
 
-### Sending a TX
+- Staking Transaction
+- Unbonding Transaction
+- Slashing Transaction
 
-## Unbonding
+### Staking Transaction
 
-### Sending a TX
+A staking transaction is different to the other transactions. It creates a UTXO
```

Comment:
I think the critical piece of infromation there here is that staking transactions must contain exactly one UTXO which commits to Babylon designed scripts. 

All other UTXOs are ignored by Babylon